packages/@lwc/engine-core/src/framework/stylesheet.ts

```diff
@@ -43,7 +44,7 @@ function getCachedStyleElement(styleContent: string): HTMLStyleElement {
     return fragment.cloneNode(true).firstChild as HTMLStyleElement;
 }
 
-const globalStyleParent = document.head || document.body || document;
+const globalStyleParent = documentObject.head || documentObject.body || documentObject;
```

Comment:
The entire stylesheet.ts to be rewritten to accommodate `@lwc/engine-server`. This reference will quickly go away.

Reply:
that's fine! baby steps!

Reply:
The sister PR can be found here: https://github.com/salesforce/lwc/pull/1926